src/FeatureTransforms.jl

```diff
@@ -7,7 +7,8 @@ using Tables
 
 export Transform, transform, transform!
 export HoD, LinearCombination, OneHotEncoding, Periodic, Power
-export AbstractScaling, IdentityScaling, MeanStdScaling
+export AbstractScaling, IdentityScaling
+export MeanStdScaling, LogTransform, InverseHyperbolicSine
```

Comment:
```suggestion
export AbstractScaling, IdentityScaling, MeanStdScaling
export LogTransform, InverseHyperbolicSine
```
so that linear scalings are together